keps/sig-network/20190603-endpointslices/README.md

```diff
@@ -626,6 +678,173 @@ address types.
 3. Deprecate the `IP` address type, making it invalid for new EndpointSlices in
 1.17 before becoming fully invalid in 1.18.
 
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - *EndpointSlice*: Guards controller and API server enablement.
+    - *EndpointSliceProxying*: Guards kube-proxy enablement
+
+* **Does enabling the feature change any default behavior?**
+  No.
```

Comment:
For the first, I agree.

For the second, assuming no bugs that's true. But in case of bugs, kube-proxy is programmed differently, so some things may change. Let's mention that explicitly.

Reply:
Thanks, added a not about what kube-proxy bugs could visibly break.